### ISSUE TITLE
Confluence integration 

### ISSUE THREAD

**Issue by ayo-yemi** [ParabolInc/parabol#8062]
## Issue - Enhancement

A [User ](https://app.hubspot.com/contacts/3888472/contact/60782151) 🔒 has requested integration with Confluence as the engineers at their company use Confluence frequently.  

### Acceptance Criteria (optional)
Until we have acceptance criteria and estimated effort, we will keep this issue open and collect more feedback.

**Comment by gcrickman** [comment]
We had 4 users request this from the PIO board and another Enterprise Trial [user](https://app.hubspot.com/contacts/3888472/record/0-1/180523503)🔒 request it today.

> I have a quick question: Is there a way for us to integrate with confluence?

**Comment by Dschoordsch** [comment]
@gcrickman do they just want to push the meeting summary to confluence or which level of integration do they expect?

**Comment by wotta** [comment]
I'm also missing a confluence integration. Reason for this is because a lot of companies do use Atlassian products including Confluence to keep records because it is company policy.
In the most simple sense it would be a nice if an integration exists that would use the "retrospective" template and fill all details. While I do understand it can be quite a lot of work I would like to ask if we could get a review on this feature request since favoring Notion feels to easy.

Are there implementation issues making this really hard to implement or would this be something that Parabol wouldn't want to maintain? And if the maintaining is the issue are there plans to maybe allow "third-party" extensions making this possible?

**Comment by Dschoordsch** [comment]
Do I understand this correctly: You want to run a retrospective on Parabol as you do now and after the meeting ended and you got the summary, you want to also store the data from the summary in a structured way in a custom template in Confluence?

**Comment by wotta** [comment]
Yes, while it is not a custom template. It's the default one that Confluence provides.

I understand it may sounds completely useless but it is something that I've seen at multiple companies. Hope this helps!

**Comment by charlesmathias** [comment]
A [user](https://app.hubspot.com/contacts/3888472/record/0-1/37743821888) 🔒 requested Confluence integration for sharing meeting results internally more easily.

**Comment by jordanh** [comment]
Just as this was subject to being iceboxed, I got a passionate request for this integration today from Black Label, a group that works with 90th COS via `{P1}`.

It might not be difficult to use text-transformed version of a retrospective for the Confluence integration content... it might be worth experimenting with during a cooldown...
